android/src/main/java/com/swmansion/reanimated/NativeMethodsHelper.java

```diff
@@ -55,6 +55,13 @@ public static void scrollTo(View view, double argX, double argY, boolean animate
       }
     }
 
+    // When we use useScrollValueOffset hook, we don't know the direction of scrolling
+    if (Double.isInfinite(argX)) {
+      int scrollValue = Math.min(x, y);
```

Comment:
Since `x` is infinity, `y` is always less than `x`, so can we use `y` directly instead?